bindings/rust/bench/benches/handshake.rs

```diff
@@ -16,16 +18,27 @@ fn bench_handshake_for_library<T: TlsConnection>(
     kx_group: KXGroup,
     sig_type: SigType,
 ) {
+    // make configs before to save time
```

Comment:
nit: I don't think we're explicitly doing this to save time. Guidance for these libraries is to reuse configs, so that's what we think customers are doing, so that's what we should be benchmarking.

Reply:
Makes sense, in the context of flamegraphs that's what I was going for, but it's also just something we should've been doing.